tests/regress/x86_self_modifying.s

```diff
@@ -0,0 +1,51 @@
+.intel_syntax noprefix
```

Comment:
a minor thing: would you please add some instructions on how to compile it into this file?